cli/list_cmd.ml

```diff
@@ -0,0 +1,138 @@
+open Import
+open Duniverse
+
+type t = {
+  name : string;
+  version : string;
+  loc : string;
+  pinned : bool;
+  descr : string option;
+}
+
+(* Assumes max_len was correctly computed, otherwise the input string might be
+   truncated. *)
+let pad s max_len =
```

Comment:
I think that this can be expressed as:

```ocaml
Printf.sprintf "%-*s" max_len s
```

(this also assumes that `max_len` is correct - if `s` is larger it's going to overflow)

Reply:
That's neat! A bit cryptic I think but I guess the wrapping function name should be enough.